Fixed the code and made it working again

suyashbansal opened the pull request:
Did some drastic changes in the core functions and add lots of comments while teaching myself :satisfied:

kingster commented on way2sms-api.php:
```diff
@@ -21,12 +21,13 @@ function sendWay2SMS($uid, $pwd, $phone, $msg)
 
-  $autobalancer = rand(1, 8);
+  //$autobalancer = rand(1, 8);
+  $autobalancer = 23;
```
This will lead to all the request's going to one of way2sms server, and if that server is loaded, this may cause sms to be delivered late. 

Change this back to a random range of valid way2sms servers :)

suyashbansal commented on way2sms-api.php:
```diff
@@ -67,25 +77,18 @@ function sendWay2SMS($uid, $pwd, $phone, $msg)
-    // Send SMS
-    curl_setopt($curl, CURLOPT_URL, 'http://site'.$autobalancer.'.way2sms.com/quicksms.action');
+    // Setup to send SMS
+    curl_setopt($curl, CURLOPT_URL, 'http://site'.$autobalancer.'.way2sms.com/smstoss.action');
```
I tried that already. If I change it to get a random number lets say it gets anything between 10-19, it gets redirected to a different load balancer. And when I further access it at this step, the sessions gets invalid. What to do?

suyashbansal replied:
Also these are number that were mainly redirected to: 21, 24, 3, 22, 23, 25.

kingster replied:
Let me give it a try, and I will get back to you!

suyashbansal commented:
So I changed the whole process of load balancer. Instead of generating a random value, I let the website decide where to go. I also added a unit test which check for it.
